Formula/openjdk.rb

```diff
@@ -147,15 +148,16 @@ def install
         cp_r "#{framework_path}/JavaNativeFoundation.framework", dest, remove_destination: true
 
         # Replace Apple signature by ad-hoc one (otherwise relocation will break it)
-        system "codesign", "-f", "-s", "-", "#{dest}/Versions/A/JavaNativeFoundation"
+        system "codesign", "-f", "-s", "-", dest/"Versions/A/JavaNativeFoundation"
       end
     end
 
     on_linux do
       libexec.install Dir["build/linux-x86_64-server-release/images/jdk/*"]
-      bin.install_symlink Dir["#{libexec}/bin/*"]
-      include.install_symlink Dir["#{libexec}/include/*.h"]
-      include.install_symlink Dir["#{libexec}/include/linux/*.h"]
+      bin.install_symlink Dir[libexec/"bin/*"]
+      include.install_symlink Dir[libexec/"include/*.h"]
+      include.install_symlink Dir[libexec/"include/linux/*.h"]
+      man.install_symlink Dir[libexec/"man/*"]
```

Comment:
CI failure probably due to nested directory structure
```
Could not symlink share/man/man1
```

One option would be to identify all `man*`, e.g.
```suggestion
      man1.install_symlink Dir[libexec/"man/man1/*"]
```

Or perhaps do a loop on all subdirs `Dir[libexec/"man/*"].each do |man|`...